### ISSUE TITLE
API Timesheet/Post -> billable=false not processed

### ISSUE THREAD

**Issue by KatjaGlassConsulting** [kevinpapst/kimai2#3254]
When using the API for timesheet post, the billable=false is not applied (version 1.19.2, it is working on 1.16.10).

**To Reproduce**

Request header:

activity: "23"
begin: "2022-04-07 12:45"
**billable: false**
description: ""
end: "2022-04-07 17:00"
project: "20"

Response:

activity: 23
begin: "2022-04-07T12:45:00+0200"
**billable: true**
description: null
duration: 15300
end: "2022-04-07T17:00:00+0200"
exported: false
fixedRate: null
hourlyRate: 0
id: 314
internalRate: 0
metaFields: []
project: 20
rate: 0
tags: []
user: 14

**Additional context**
 - Kimai version: 1.19.2 
 - PHP version: 7.4.25

**Comment by kevinpapst** [comment]
Yeah, you might or might not have seen the changes regarding the auto calculation of the billable field.
Seems this has caused a BC break.
I don't have time right now, maybe you can debug it.  Here are two PRs: 
- the original change https://github.com/kevinpapst/kimai2/pull/3200
- and a fix for another issue https://github.com/kevinpapst/kimai2/pull/3225

Maybe this overwritten method is the cause: 
https://github.com/kevinpapst/kimai2/blob/master/src/Form/API/TimesheetApiEditForm.php#L20

What happens when you remove it and let the base method take over:
https://github.com/kevinpapst/kimai2/blob/master/src/Form/TimesheetEditForm.php#L264
?

**Comment by KatjaGlassConsulting** [comment]
I am more a js and react developer and less a php developer. I might be able to checkout this out in the week of 18.04. Thanks for pointing out the likely problematic area.
